doc/man/8/ceph-fuse.rst

```diff
@@ -64,4 +64,4 @@ See also
 fusermount(8),
 :doc:`ceph <ceph>`\(8)
 
-.. _Mount Ceph FS using FUSE: ../../cephfs/fuse/
+.. _Mount Ceph FS using FUSE: ../../../cephfs/fuse/
```

Comment:
I'm not familiar with doc, what is this line used for (I don't see it when viewing this doc)?

Reply:
It is for providing url for `Mount Ceph FS using FUSE`  in the below sentence. You can see it in the [raw](https://raw.githubusercontent.com/joscollin/ceph/5cd3b8c759c48a23c9ba04d8ba6a373c0f462320/doc/man/8/ceph-fuse.rst).

See Mount Ceph FS using FUSE for detailed information.

Reply:
Now the problem is that it is missing a `..` and the URL doesn't work as expected.